sormas-api/src/main/java/de/symeda/sormas/api/Disease.java

```diff
@@ -24,76 +24,78 @@
+	CONGENITAL_RUBELLA(true, true, true, true, 21, true),
+	CSM(true, true, true, false, 10, true),
+	DENGUE(true, true, true, false, 14, true),
+	EVD(true, true, true, true, 21, true),
+	GUINEA_WORM(true, true, true, false, 0, true),
+	LASSA(true, true, true, true, 21, true),
+	MEASLES(true, true, true, false, 21, true),
+	MONKEYPOX(true, true, true, true, 21, true),
+	NEW_INFLUENZA(true, true, true, true, 17, true),
+	PLAGUE(true, true, true, true, 7, true),
+	POLIO(true, true, true, false, 0, true),
+	UNSPECIFIED_VHF(true, true, true, true, 21, true),
+	WEST_NILE_FEVER(true, false, true, false, 0, true),
+	YELLOW_FEVER(true, true, true, false, 6, true),
+	RABIES(true, true, true, true, 6, true),
+	ANTHRAX(true, true, true, false, 0, true),
+	CORONAVIRUS(true, true, true, true, 14, true),
+	PNEUMONIA(true, false, true, false, 0, true),
+	MALARIA(true, false, false, false, 0, true),
+	TYPHOID_FEVER(true, false, false, false, 0, true),
+	ACUTE_VIRAL_HEPATITIS(true, false, false, false, 0, true),
+	NON_NEONATAL_TETANUS(true, false, false, false, 0, true),
+	HIV(true, false, false, false, 0, true),
+	SCHISTOSOMIASIS(true, false, false, false, 0, true),
+	SOIL_TRANSMITTED_HELMINTHS(true, false, false, false, 0, true),
+	TRYPANOSOMIASIS(true, false, false, false, 0, true),
+	DIARRHEA_DEHYDRATION(true, false, false, false, 0, true),
+	DIARRHEA_BLOOD(true, false, false, false, 0, true),
+	SNAKE_BITE(true, false, false, false, 0, true),
+	RUBELLA(true, false, false, false, 0, true),
+	TUBERCULOSIS(true, false, false, false, 0, true),
+	LEPROSY(true, false, false, false, 0, true),
+	LYMPHATIC_FILARIASIS(true, false, false, false, 0, true),
+	BURULI_ULCER(true, false, false, false, 0, true),
+	PERTUSSIS(true, false, false, false, 0, true),
+	NEONATAL_TETANUS(true, false, false, false, 0, true),
+	ONCHOCERCIASIS(true, false, false, false, 0, true),
+	DIPHTERIA(true, false, false, false, 0, true),
+	TRACHOMA(true, false, false, false, 0, true),
+	YAWS_ENDEMIC_SYPHILIS(true, false, false, false, 0, true),
+	MATERNAL_DEATHS(true, false, false, false, 0, true),
+	PERINATAL_DEATHS(true, false, false, false, 0, true),
+	INFLUENZA_A(true, false, true, false, 0, true),
+	INFLUENZA_B(true, false, true, false, 0, true),
+	H_METAPNEUMOVIRUS(true, false, true, false, 0, true),
+	RESPIRATORY_SYNCYTIAL_VIRUS(true, false, true, false, 0, true),
+	PARAINFLUENZA_1_4(true, false, true, false, 0, true),
+	ADENOVIRUS(true, false, true, false, 0, true),
+	RHINOVIRUS(true, false, true, false, 0, true),
+	ENTEROVIRUS(true, false, true, false, 0, true),
+	M_PNEUMONIAE(true, false, true, false, 0, true),
+	C_PNEUMONIAE(true, false, true, false, 0, true),
+	OTHER(true, true, true, true, 21, false),
+	UNDEFINED(true, true, true, true, 0, false);
 
 	private boolean defaultActive;
 	private boolean defaultPrimary;
 	private boolean defaultCaseBased;
 	private boolean defaultFollowUpEnabled;
 	private int defaultFollowUpDuration;
+	private boolean allowVariant;
```

Comment:
Rename to `variantAllowed` (to make it consistent with the setter)

Reply:
Done